### PR TITLE
chore: update supported go versions

### PR DESCRIPTION
**Pull request by gssbzn**
## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by andreaangiolillo — APPROVED** [review]
LGTM! Just one thing: when I click on the link "Continue to review full report at Codecov." I got this error. Is this normal?
Screenshot 2020-09-02 at 13 26 41

**Comment by gssbzn** [comment]
The problem is because we don't have code coverage on master so once this is merge we'll get it

**Pull request merged**
